orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/RunJobStage.java

```diff
@@ -16,20 +16,40 @@
 
 package com.netflix.spinnaker.orca.clouddriver.pipeline.job;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.orca.CancellableStage;
 import com.netflix.spinnaker.orca.clouddriver.tasks.artifacts.ConsumeArtifactTask;
+import com.netflix.spinnaker.orca.clouddriver.tasks.job.DestroyJobTask;
 import com.netflix.spinnaker.orca.clouddriver.tasks.job.MonitorJobTask;
 import com.netflix.spinnaker.orca.clouddriver.tasks.job.RunJobTask;
 import com.netflix.spinnaker.orca.clouddriver.tasks.job.WaitOnJobCompletion;
+import com.netflix.spinnaker.orca.jackson.OrcaObjectMapper;
 import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder;
 import com.netflix.spinnaker.orca.pipeline.TaskNode;
 import com.netflix.spinnaker.orca.pipeline.model.Stage;
 import com.netflix.spinnaker.orca.pipeline.tasks.artifacts.BindProducedArtifactsTask;
 import java.util.HashMap;
 import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
-public class RunJobStage implements StageDefinitionBuilder {
+public class RunJobStage implements StageDefinitionBuilder, CancellableStage {
+
+  private final Logger log = LoggerFactory.getLogger(getClass());
+  private final DestroyJobTask destroyJobTask;
+  private final ObjectMapper objectMapper = OrcaObjectMapper.newInstance();
+
+  public RunJobStage() {
```

Comment:
why is this needed? I suspect it is just so existing tests don't need to be updated... I'd lean towards just updating the tests to pass in a stub if that is the case

Reply:
You are correct, I'll fix the tests so this is not needed.